website/next-env.d.ts

```
@@ -1,5 +1,6 @@
 /// <reference types="next" />
 /// <reference types="next/image-types/global" />
+/// <reference path="./.next/types/routes.d.ts" />
```

ESLint

[error] Do not use a triple slash reference for ./.next/types/routes.d.ts, use `import` style instead.

(@typescript-eslint/triple-slash-reference)